### PR TITLE
refactor: changed React.PropTypes to standalone prop-types package

### PR DESCRIPTION
**Pull request by hafuta**
refactor: changed React.PropTypes to standalone prop-types package, added yarn.lock file

### CODE REVIEW

**Comment by oayres** [comment]
@d3viant0ne @sairion @kinetifex Sorry to directly tag you guys, but is there any chance this can be merged and published to npm please?

File: lib/component.jsx [hunk 1]
```diff
@@ -16,8 +16,8 @@ var IconSVG = React.createClass({
         };
     },
```
Comment (d3viant0ne) [inline lib/component.jsx]: This entire block is deprecated & removed in #55.

**Comment by hafuta** [comment]
@d3viant0ne removed yarn.lock

**Comment by hafuta** [comment]
Closing

**Pull request closed without merging**